test/test_cli.py

```diff
@@ -25,7 +25,7 @@ def test_parsing_config_file(test_arguments):
 def test_parsing_opensuse_conf(test_arguments):
     parsed = process_lint_args(test_arguments)
 
-    assert len(parsed['config']) == 13
+    assert len(parsed['config']) == 12
```

Comment:
Is it really a good idea to test for the exact number of entries of the production config? This test will surely fail every time somebody changes something in the configuration and cause a lot of noise and trouble.

Reply:
@marxin

Reply:
Yes, please remote it, it's pretty fragile as it seems.